ui/src/flux/components/DatabaseListItem.tsx

```diff
@@ -83,6 +81,44 @@ class DatabaseListItem extends PureComponent<Props, State> {
     })
   }
 
+  private get filterAndTagList(): JSX.Element {
+    const {db, service} = this.props
+    const {isOpen, searchTerm} = this.state
+
+    if (isOpen) {
+      return (
+        <>
```

Comment:
any idea what's going on with this empty arrow brackets? i see it's in the original code, not introduced by this PR

Reply:
In React 16 you don't have to wrap a div around what you are returning if there are multiple components being returned. It's the shorthand syntax for react fragments.

Reply:
very cool -- thanks!